### DISCUSSION TITLE
To change Date and Time Formate in log file

### DISCUSSION THREAD

**Discussion by Sivanesh1992** [herlesupreeth/docker_open5gs#239] [General]
hi @herlesupreeth  i need to change date and time format in each core network logs.

As i seen after taking core network log  format is  MM/Date HH:MM:SEC.806:

I need to change [YYYY-MM-DATE  HH:MM:SEC.298]   to this format.

is this possible?

**Comment by herlesupreeth** [comment]
I dont think its possible unless you modify the open5gs code
